standup/status/jinja2/base.html

```diff
@@ -91,10 +91,9 @@
 
   <div id="main-content" class="container_12 cf">
     <div id="main-notices" class="grid_12">
-      <div class="notice success">
-        We've switched the way we use Auth0. If you have problems, see
-        <a href="{{ settings.HELP_FAQ_URL }}">our FAQ</a>.
-      </div>
+      {% for message in get_messages_for_page(request) %}
+        <div class="notice success">{{ message }}</div>
```

Comment:
I used "success" because it was green and I didn't want to alarm anyone. I don't know if we want all our notices to be success ones. I don't know if it matters a whole lot, but if messages had other levels, that might be good to expose.

Reply:
I take it back--I don't see anything about levels in django-soapbox.

Reply:
I was hoping it had some concept of `tag` or `class` or something we could use as a CSS class as well. Could roll our own or fork soapbox for that, but I decided for now it's not worth it.

Reply:
We could change it to `info`, which I think is blue, so might be more appropriate for announcements. I was just sticking to using what had been working for now.